nix/flakebox.nix

```diff
@@ -610,6 +610,9 @@ rec {
           pkgs.coreutils
           pkgs.sqlite
           pkgs.fakeNss
+          pkgs.busybox
```

Comment:
I wonder if using `busybox` pkg can remove the need for `fakeNss`, need to take a look at it.